packages/firestore/test/unit/specs/perf_spec.test.ts

```diff
@@ -66,7 +66,7 @@ describeSpec(
               fromCache: true,
               hasPendingWrites: true
             })
-            .writeAcks(`collection/${i}`, ++currentVersion)
+            .writeAcks(`collection/${i}`, docRemote.version.toMicroseconds())
             .watchAcksFull(query, ++currentVersion, docRemote)
```

Comment:
Note that in the existing code, this watchAck will get ignored since docRemote.version here is greater than the writeAck version.